starport/pkg/cosmosgen/generate_javascript.go

```diff
@@ -197,7 +197,13 @@ func (g *jsGenerator) generateModule(ctx context.Context, tsprotoPluginPath, app
 			return err
 		}
 	}
-
+	// generate package.json for Vuex if enabled.
+	if g.g.o.vuexStoreRootPath != "" {
```

Comment:
can we move the content of this block to the prev if statement, because they have the same condition.